src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const items = document.querySelectorAll('li');
+
+function sortList() {
+  return [...items].sort((a, b) => (
+    +b.dataset.salary.replace(/[^\d]/g, '')
+    - +a.dataset.salary.replace(/[^\d]/g, '')
+  ));
+}
+
+function getEmployees() {
+  const sortedEmployees = sortList(items);
```

Comment:
you pass an argument to a function but your function doesn't accept this argument